misc/dbt-materialize/tests/adapter/test_custom_materializations.py

```diff
@@ -47,13 +48,15 @@ def models(self):
             "actual_indexes.sql": actual_indexes,
             "test_materialized_view.sql": test_materialized_view,
             "test_materialized_view_index.sql": test_materialized_view_index,
+            "test_materialized_view_nullability_assertions.sql": test_materialized_view,
```

Comment:
Is there a way we can test that the `ASSERT NOT NULL` clauses are actually getting installed into Materialize?

Reply:
Best I could come up with is running a `show create` on the materialized view and checking if the sql contains the assertion section.

Reply:
Nice, that seems like a great approach!